moveit_planners/chomp/chomp_interface/package.xml

```diff
@@ -20,7 +20,10 @@
   <build_depend>chomp_motion_planner</build_depend>
   <build_depend>moveit_experimental</build_depend>
 
+  <test_depend>moveit_ros_manipulation</test_depend>
   <test_depend>moveit_ros_planning_interface</test_depend>
+  <test_depend>roscpp</test_depend>
```

Comment:
No need to add `roscpp` here, `test-depend` is added on top of the normal `build-depend` list.